reference-architecture/osp-dns/heat/fragments/install_python.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+#
+# Ensure that Python is installed for Ansible
+#
+echo "Installing Python, DNF and Selinux packages for Ansible"
+[ -f /etc/fedora-release ] && dnf install -y python python-dnf libselinux-python
```

Comment:
Following this recommendation for Fedora: 
https://fedoramagazine.org/getting-ansible-working-fedora-23/